packages/components/src/components/inputs/Input/index.native.tsx

```diff
@@ -45,31 +184,44 @@ const Input = ({
     wrapperProps,
     ...rest
 }: Props) => {
-    return null;
-};
+    const stateIcon = getStateIcon(state);
+    const stateColor = getPrimaryColor(state) || undefined;
 
-Input.propTypes = {
```

Comment:
proptypes 👀 let's keep them everywhere od delete them completely. I like the first option because It is not much more work to do if we already have ts types and it helps in projects without TS, but I am fine with both options.